test/test_fingerprint_api.py

```diff
@@ -737,7 +752,11 @@ def test_search_events_all_params(self):
             timeout=None
         )
 
-        response = self.api.search_events(**params)
+        multi = {}
+        for k, v in multivalue_params:
```

Comment:
You added extra logic again that makes test unobvious.
We should see what we pass and what we get in a clear way to be sure that test works.
Ye, it would be like 2 source of truth, but it is a test data and iti expected in such cases.